Mage.Sets/src/mage/cards/s/SharaeOfNumbingDepths.java

```diff
@@ -83,7 +83,6 @@ public boolean checkEventType(GameEvent event, Game game) {
     public boolean checkTrigger(GameEvent event, Game game) {
         Permanent permanent = game.getPermanent(event.getTargetId());
         return permanent != null
-                && StaticFilters.FILTER_OPPONENTS_PERMANENT_CREATURE.match(permanent, game)
-                && isControlledBy(event.getPlayerId());
```

Comment:
This fix is not correct.

The `isControlledBy(event.getPlayerId())` is necessary because it needs to only trigger when you tap, not when becomes tapped.

The root cause of the issue is two argument match on the filter ignoring ObjectSourcePlayerPredicates. It's not correct to put `event.getPlayerId()` there, that part needs to be `this.getControllerId()`.